fastchat/model/model_adapter.py

```diff
@@ -644,6 +644,13 @@ def load_model(self, model_path: str, from_pretrained_kwargs: dict):
         return model, tokenizer
 
 
+class FlanAdapter(T5Adapter):
+    """The model adapter for flan-t5-*, flan-ul2"""
+
+    def match(self, model_path: str):
+        return "flan" in model_path.lower()
+
```

Comment:
What is the instruction template for flan? Do we need to set it here?

Reply:
Hi @merrymercy, do you mean prompt template?

Reply:
yes

Reply:
@wangzhen263 Any updates? IIRC, flan-t5 has a default prompt template.